src/Symfony/Component/RateLimiter/SlidingWindow.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\RateLimiter;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * @experimental in 5.2
+ */
+final class SlidingWindow implements LimiterStateInterface
+{
+    private $id;
```

Comment:
Should we add phpdoc types for the properties used in this PR?

Reply:
I would love to. Yes